templates/activityselector.mustache

```diff
@@ -71,11 +71,12 @@
         <option value="{{id}}" disabled class="learningmap-disabled-activity" title="{{#str}} nocompletionenabled, learningmap {{/str}}">{{#shortentext}} 30, {{{name}}} {{/shortentext}}</option>
         {{/completionenabled}}
         {{#completionenabled}}
-        <option value="{{id}}" title="{{name}}">{{#shortentext}} 30, {{{name}}} {{/shortentext}}</option>
+        <option value="{{id}}" title="{{name}}" data-activity-hidden="{{hidden}}">{{#shortentext}} 30, {{{name}}} {{/shortentext}}</option>
```

Comment:
Please add the placeholder tag "hidden" above in the json example context

Reply:
Fixed!